linera-views/src/store.rs

```diff
@@ -29,6 +29,8 @@ pub struct CommonStoreConfig {
     pub max_stream_queries: usize,
     /// The cache size being used.
     pub cache_size: usize,
+    /// Caching the absence of keys.
+    pub cache_key_absence: bool,
```

Comment:
This belongs to the `LruCachingConfig`

Reply:
Actually, maybe it should be in `LruCachingStore` actually.
It should be safe to set this flag to true every time a store is returned by `<LruCachingStore as AdminKeyValueStore>::connect` and `clone_with_root_key`.